tools/create-release-announce-template-for-twitter.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -ue
+
+PGROONGA_LATEST_VERSION_AND_DATE=$(curl https://api.github.com/repos/pgroonga/pgroonga/releases/latest | jq '.name' | sed -e 's/^"//' -e 's/"$//')
+PGROONGA_LATEST_VERSION=$(curl https://api.github.com/repos/pgroonga/pgroonga/releases/latest | jq '.tag_name' | sed -e 's/^"//' -e 's/"$//')
+
+RELEASE_BLOG_BASE_URL="https://groonga.org"
+RELEASE_BLOG_DATE=$(echo $PGROONGA_LATEST_VERSION_AND_DATE | awk -F ":" '{print $2}' | sed -e 's/^ //' -e 's/"$//' | sed -e 's/-/\//g')
```

Comment:
Why do you prefer `YYYY/MM/DD` to `YYYY-MM-DD`? `YYYY-MM-DD` is the ISO 8601 format.

Reply:
Ah, you want a path.

Reply:
`RELEASE_BLOG_DATE_PATH` may be better.

Reply:
`echo $PGROONGA_LATEST_VERSION_AND_DATE | cut -d ' ' -f 3 | tr - /`